### ISSUE TITLE
Could your Nexus DL file be corrupted?

### ISSUE THREAD

**Issue by Ra1v1n** [TanninOne/modorganizer#1145]
I Just downloaded the .exe file for MO from Nexus not even an hour ago & when I installed it, the following .jpgs show what my result was. My previous install had a similar issue, but with only 2 files, and with different symbols, but I thought it was just a one-time glitch & ignored it. Just to be sure it didn't happen again, I thought I'd get a fresh file to work with. but it seems to have gotten worse, not better. The installations were on separate drives on the same computer, which is an HP Pavilion 23 All-in-one with an AMD E2-2000 APU with Radeon HD Graphics 1.75Ghz and with 4GB Ram. if that helps any. 

![mo installation 4](https://user-images.githubusercontent.com/33211602/32599369-532a5ca4-c50a-11e7-90d7-0d8c3a60605a.jpg)
![mo installation 3](https://user-images.githubusercontent.com/33211602/32599371-551ab8ec-c50a-11e7-9b43-fc152a492360.jpg)
![mo installation 2](https://user-images.githubusercontent.com/33211602/32599374-56c7185c-c50a-11e7-81de-b426014004ad.jpg)
![mo installation 1](https://user-images.githubusercontent.com/33211602/32599377-588a0b68-c50a-11e7-9515-74c3cbde03bb.jpg)

**Comment by TanninOne** [comment]
No, I just verified the files on nexus are fine and unmodified since I uploaded them.
The fact you got files like nsDialogs.dll indicates you didn't run the installer but extracted it manually since those files belong to the installer, they don't get extracted.
